float32 x_center
float32 y_center
float32 width
float32 length
float32 yaw